plugins/catalog-backend/src/service/CatalogBuilder.ts

```diff
@@ -351,19 +345,9 @@ export class CatalogBuilder {
   getDefaultProcessors(): CatalogProcessor[] {
     const { config, logger, reader } = this.env;
     const integrations = ScmIntegrations.fromConfig(config);
-    const githubCredentialsProvider: GithubCredentialsProvider =
-      DefaultGithubCredentialsProvider.fromIntegrations(integrations);
 
     return [
       new FileReaderProcessor(),
-      GithubDiscoveryProcessor.fromConfig(config, {
```

Comment:
This removal is a lil bit scary, I'm wondering if there's anything we can do to make this more obvious in case someone doesn't read the changeset? Preferably having the catalog fail to start up if misconfigured

Reply:
Same level of scary as all of the other discovery processors though. I was thinking we should make special announcements on discord about this, it warrants being called out separately. Hm but now that you mention it, maybe the `build` method could go through at least the configured locations and the names of the list of processors added, to see if things won't work at runtime. I'll start making a PR for that check (for all discovery processors at the same time) immediately